quickstart-python/routes/auth.py

```diff
@@ -0,0 +1,46 @@
+from urllib.parse import urlencode
+
+from flask import request, Blueprint, redirect
+
+from consts import SecretKeys
+from middlewares import monday_request_auth
+from services import SecureStorage, SecretService, JWTService
+
+auth_bp = Blueprint('auth', __name__)
+
+
+@auth_bp.route('/', methods=['GET'])
+@monday_request_auth
+def authorize():
+    """
+    Redirects the user to the monday.com OAuth2 authorization page.
+    First stage of the OAuth flow, Invoked when the user clicks on `Use template` for instance
+    """
+    user_id = request.session.get('userId')
+    back_to_url = request.session.get('backToUrl')
+
+    connection = SecureStorage.get(user_id)
+    if connection and connection['monday_token']:
+        return redirect(back_to_url)
+
+    SecureStorage.put(user_id, {'back_to_url': back_to_url})
+
+    params = {'client_id': SecretService.get_secret(SecretKeys.MONDAY_OAUTH_CLIENT_ID), 'state': user_id}
+    redirect_url = f"{SecretService.get_secret(SecretKeys.MONDAY_OAUTH_BASE_PATH)}?{urlencode(params)}"
```

Comment:
MONDAY_OAUTH_BASE_PATH is not a secret IMO, it fits more to be an env variable

Reply:
There's no env variables manager in the monday_code library, only secrets, which takes them from the environment variable in the monday code UI. So it's essentially the same no?

Reply:
You are right that ATM we don't have it, but @maorb-dev is now working on adding it to the SDK so that we have 2 types of in-mem key-value:
1. Env variables - they are loaded into process.env and also available through the [EnvironmentVariablesManager](https://developer.monday.com/apps/docs/monday-code-sdk#environment-variables-manager) - this is the one you are using here (it should not be `SecretService`, but `EnvironmentVariablesService`)
2. Secrets - this **WILL** be available soon (@maorb-dev I'm looking at you 😄) and also will be renamed in the SDK (so you will need to change it)

Reply:
Ohhh got it. so for now there's no reason to rename it right?